admin-ui/plugins/fido/helper/constants.ts

```diff
@@ -35,10 +35,10 @@ export const fidoConstants = {
     MDS_TOCS_FOLDER: 'mdsTocsFolder',
     CHECK_U2F_ATTESTATIONS: 'checkU2fAttestations',
     UNFINISHED_REQUEST_EXPIRATION: 'unfinishedRequestExpiration',
-    AUTHENTICATION_HISTORY_EXPIRATION: 'authenticationHistoryExpiration',
+    METADATA_REFRESH_INTERVAL: 'metadataRefreshInterval',
     SERVER_METADATA_FOLDER: 'serverMetadataFolder',
     USER_AUTO_ENROLLMENT: 'userAutoEnrollment',
-    REQUESTED_CREDENTIAL_TYPES: 'requestedCredentialTypes',
+    ENABLED_FIDO_ALGORITHMS: 'enabledFidoAlgorithms',
     REQUESTED_PARTIES: 'requestedParties',
   },
```

Comment:
Align form field and payload mappings to `rp`  
- In `constants.ts`, rename `REQUESTED_PARTIES: 'requestedParties'` to `RP: 'rp'`  
- In `helper/utils.ts` mapping (initial values), return `rp` instead of `requestedParties`  
- In `createFidoConfigPayload`, replace `payload.fido2Configuration.requestedParties = …` with `payload.fido2Configuration.rp = data.rp.map(...)`  
- Update all component references (`formik.values.requestedParties`, `fidoConstants.FORM_FIELDS.REQUESTED_PARTIES`) to use `rp` so that `Fido2Configuration.rp` is correctly populated at both read and write stages